README.md

```diff
@@ -254,7 +254,7 @@ Application Options:
       --tlsv1.2                                             Forces gocurl to use TLS v1.2.
       --http1.1                                             Forces gocurl to use HTTP v1.1.
       --http2                                               Forces gocurl to use HTTP v2.
-      --http3                                               Forces gocurl to use HTTP v2.
+      --http3                                               Forces gocurl to use HTTP v3.
```

Comment:
Could you please also fix this typo in options.go? It comes from there

Reply:
fixed